.travis.yml

```diff
@@ -1,14 +1,22 @@
 sudo: required
 
-language: bash
-
 services:
   - docker
 
+env:
+  -
```

Comment:
Chvili mi trvalo nez mi to v kombinaci s commentem v `Dockerfile` doslo. Hodil bych sem comment, ze je to umyslne, jinak to urcite nekdo pri prvni prilezitosti odstrani jako preklep :)